lib/lambda/update/updatePackage.ts

```diff
@@ -0,0 +1,127 @@
+import { response } from "libs/handler-lib";
+import { APIGatewayEvent } from "aws-lambda";
+import { getPackage } from "libs/api/package";
+import { produceMessage } from "libs/api/kafka";
+
+type ActionType = "update-id" | "update-values" | "delete";
+
+export const handler = async (event: APIGatewayEvent) => {
+  const topicName = process.env.topicName as string;
+
+  if (!topicName) {
+    throw new Error("Topic name is not defined");
+  }
+
+  if (!event.body) {
+    return response({
+      statusCode: 400,
+      body: { message: "Event body required" },
+    });
+  }
+  try {
+    // TODO: allow user to input title of the accordion
+    const { packageId, action, updatedFields, changeReason } =
+      typeof event.body === "string"
+        ? JSON.parse(event.body)
+        : (event.body as {
+            packageId: string;
+            action: ActionType;
+            updatedFields: object;
+          });
+
```

Comment:
Would it be better to use zod here? Define a schema in the function and use it to safely parse `event.body`? Kinda like what we do with events. 

Ask Brian if that's a good idea

Reply:
Sounds good